portal/server/routes/contract/loan/index.js

```diff
@@ -94,7 +94,7 @@ const loanGuaranteeDetailsPayloadProperties = [
 ];
 
 const filterLoanGuaranteeDetailsPayload = (body) => {
-  const payload = constructPayload(body, loanGuaranteeDetailsPayloadProperties);
+const payload = constructPayload(body, loanGuaranteeDetailsPayloadProperties,true);
```

Comment:
It will insert the empty properties but when we insert the value, the property will update and code will check the validation also for all the pages.

Reply:
ConstructPayload function can expect canPropertyBeEmpty variable. Default the variable is false instead of false we passed true. So constructPayload function will construct the payload variables with empty value. That payload will save in DB object. Second time we come to the same page we fetch all the details from DB for the page and check the value is exist or not. If value is null then will display the error otherwise no error will display. When we enter the value for the empty object, the post will construct new payload and update the DB.